index.bs

```diff
@@ -842,7 +842,7 @@ To <dfn>sanitize a document fragment</dfn> named |fragment| with a {{Sanitizer}}
     1. [=map/Set=] |m|[|node|] to |action|.
   1. [=list/iterate|For each=] |node| in |nodes|:
     1. If |m|[|node|] is `drop`, [=/remove=] |node|.
-    1. If |m|[|node|] is `block`, [=/replace=] |node| with all of its element and text node children from |fragment|.
+    1. If |m|[|node|] is `block`, invoke [=Replace with=] with |node| as |this| and |node|'s [=tree/children=] as |nodes|.
```

Comment:
Hmm. As I read `[=/replace=]`, it explicitly handles the case of a DocumentFragment (and does the right thing for us, too). I think just dropping the "with all of its element and text node children" language and just using |fragment| as |node| should be correct.

`[=ReplaceWith=]` -- if I found the correct `replaceWith`, which I'm not sure about -- calls into `converting nodes into a node`, which creates a DocumentFragment, and then calls `replace` with it. If so, then this would just add extra steps by taking the nodes out of a fragment; then putting them into a new fragment; and then calling replace, which takes the nodes out of their new fragment again and puts them into the DOM. Technically correct, but arguably not easier to understand.

---

General question: Is there a convenient method to resolve these spec references? Best I can think of right now is patching in the change locally; running bikeshed; and then seeing where it takes me.

Reply:
Well, invoking `replace` without a parent reference would work just how `[=replaceWith=]` does it, by `let ... be |node|'s [=parent=]`.

I now notice that the use of `|fragment|` in the current text is plainly wrong, however, and this indeed needs to be the `[=tree/children=]` of `|node|`. And that requires creating a separate DocumentFragment anyway, as `[=replaceWith=]` implicitly does. So... let's go with `[=replaceWIth=]`.